src/templates/candidate.js

```diff
@@ -38,37 +42,64 @@ function ChartSection({ title, type, total, data, ...passProps }) {
   )
 }
 
-// TODO (#56) Create actual layout for this page
-export default function Candidate() {
+export default function Candidate({ data }) {
+  const { Name: name, Elections: elections } = data.candidate
+  // Let's assume that the first election in the list is the relevant (current) one
+  // TODO We need an error state if there are no elections
+  const { ElectionTitle: title } = elections[0]
   return (
-    <div className={styles.container}>
-      <section>
-        <SectionHeader title="Fundraising totals" />
-        <div className={styles.totals}>
-          <TotalAmountPanelItem type="contributions" total={654876} />
-          <TotalAmountPanelItem type="expenditures" total={383254} />
-          <TotalAmountPanelItem type="balance" total={271622} />
+    <Layout windowIsLarge={useWindowIsLarge()}>
+      <LandingPageHero background="blue" title={name} subtitle={title} />
+      <div className={styles.container}>
+        <div className={styles.sideNav}></div>
+        <div className={styles.mainSection}>
+          <section>
+            <SectionHeader title={name} />
+            <div className={styles.aboutSection}>
+              <p>TODO - candidate about section</p>
+            </div>
+          </section>
+          <section>
+            <SectionHeader title="Fundraising totals" />
+            <div className={styles.totals}>
+              <TotalAmountPanelItem type="contributions" total={654876} />
+              <TotalAmountPanelItem type="expenditures" total={383254} />
+              <TotalAmountPanelItem type="balance" total={271622} />
+            </div>
+          </section>
+          <ChartSection
+            title="Where the money is coming from"
+            type="contributions"
+            total={654876}
+            data={contributions}
+          />
+          <ChartSection
+            title="How the money is being spent"
+            type="expenditures"
+            total={383254}
+            data={expenditures}
+          />
+          <ChartSection
+            title="Breakdown by region"
+            type="contributions"
+            total={654876}
+            data={breakdowns}
+            showPercentages
+          />
         </div>
-      </section>
-      <ChartSection
-        title="Where the money is coming from"
-        type="contributions"
-        total={654876}
-        data={contributions}
-      />
-      <ChartSection
-        title="How the money is being spent"
-        type="expenditures"
-        total={383254}
-        data={expenditures}
-      />
-      <ChartSection
-        title="Breakdown by region"
-        type="contributions"
-        total={654876}
-        data={breakdowns}
-        showPercentages
-      />
-    </div>
+      </div>
+    </Layout>
   )
 }
+
+export const query = graphql`
+  query($slug: String!) {
+    candidate(fields: { slug: { eq: $slug } }) {
```

Comment:
Using the page slug to look up the candidate now because the /candidates GET API doesn't provide the id - not sure if that's an oversight, since the POST API does?

Reply:
Yeah, I only see a name and election data as well. @geleazar1000111 @rwalek668 Can you confirm that we're giving each candidate an ID?

Reply:
Yes, I believe every candidate should receive an ID when we populate the database. Based on the API shape right now we're not including the ID in the GET info, but we can definitely put it in.

API shape: https://docs.google.com/document/d/1pcSLUKkdn0aRsr5ml45007-bAVt40E0jdizLT0E-P0Q/edit